### PR TITLE
Converted all the css into Tailwind css of Drawer component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Converted all the css into Tailwind CSS of Drawer component.
issue - #1725 

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Drawer/Drawer.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { node, string, bool } from 'prop-types';
 import classNames from 'classnames';
-import styles from './Drawer.module.css';
```
Comment (kylemh) [inline components/Drawer/Drawer.js:3]: Can you delete the `Drawer.module.css` file, too?

**Comment by subhajit20** [comment]
@kylemh okay will do that.

File: components/Drawer/Drawer.js [hunk 2]
```diff
@@ -17,11 +16,13 @@ function Drawer({ children, className, isVisible }) {
   return (
     <div
       className={classNames(className, {
-        [styles.visible]: isVisible,
-        [styles.hidden]: !isVisible,
+        '-left-0 hidden lg:block transition-all ease-in-out duration-1000 fixed top- bottom-0  overflow-hidden width-full z-[2]':
+          isVisible,
+        '-left-[100%] hidden lg:block transition-all ease-in-out duration-1000 fixed top- bottom-0  overflow-hidden width-full z-[2]':
```
Comment (kylemh) [inline components/Drawer/Drawer.js:21]:
```suggestion
        '-left-0':
          isVisible,
        '-left-[100%]':
```

File: components/Drawer/Drawer.js [hunk 3]
```diff
@@ -17,11 +16,13 @@ function Drawer({ children, className, isVisible }) {
   return (
     <div
       className={classNames(className, {
```
Comment (kylemh) [inline components/Drawer/Drawer.js:18]:
```suggestion
      className={classNames(className,
      'hidden lg:block transition-all ease-in-out duration-1000 fixed top-0 bottom-0 overflow-hidden width-full z-[2]', {
```

**Review by kylemh — COMMENTED** [review]
Try this. Also, can you test your changes by starting the local dev server for the app? Right now, it's not visible.

File: components/Drawer/Drawer.js [hunk 4]
```diff
@@ -15,12 +15,14 @@ Drawer.defaultProps = {
...
     <div
-      className={classNames(className, {
-        '-left-0 hidden lg:block transition-all ease-in-out duration-1000 fixed top- bottom-0  overflow-hidden width-full z-[2]':
-          isVisible,
-        '-left-[100%] hidden lg:block transition-all ease-in-out duration-1000 fixed top- bottom-0  overflow-hidden width-full z-[2]':
-          !isVisible,
-      })}
+      className={classNames(
+        className,
+        'hidden lg:block transition-all ease-in-out duration-1000 fixed top- bottom-0  overflow-hidden width-full z-[2]',
```
Comment (kylemh) [inline components/Drawer/Drawer.js:20]:
```suggestion
        'hidden lg:block transition-all ease-in-out duration-1000 fixed top-0 bottom-0 overflow-hidden width-full z-[2]',
```

**Review by kylemh — APPROVED** [review]

**Pull request merged**